TypeCobol/Compiler/Directives/TypeCobolOptions.cs

```diff
@@ -32,6 +32,14 @@ public bool UseEuroInformationLegacyReplacingSyntax
             get { return _useEuroInformationLegacyReplacingSyntax; }
             set { _useEuroInformationLegacyReplacingSyntax = value; }
         }
+        /// <summary>
+        /// Are these Options used for parsing a Copy.
+        /// </summary>
+        public bool AreForCopyParsing
+        {
+            get;
+            set;
+        }
```

Comment:
I'm not sure if it is a new property in `TypeCobolOptions`, why not introduce an additional parameter in scanning method ?